_code-samples/claim-payment-channel/py/claim_pay_channel.py

```diff
@@ -0,0 +1,83 @@
+"""
+Create, claim and verify a Payment Channel.
+Reference: https://xrpl.org/paychannel.html
```

Comment:
We should use the new URL here instead of the old (redirected) one.

```suggestion
Reference: https://xrpl.org/docs/references/protocol/ledger-data/ledger-entry-types/paychannel
```